src/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cncjs-app",
-  "version": "1.9.26",
+  "version": "1.9.27",
```

Comment:
Same as noted above - apologies, I failed the version update procedure.

Reply:
It's ok. I will proceed to update the version field as per usual. 😃